src/main/resources/reference.conf

```diff
@@ -1300,6 +1290,17 @@ otarproject {
 }
 
 literature {
+
+  // Bucket name information
+  bucketName = "open-targets-pre-data-releases"
+  bucketPathBase = "gs://"${literature.bucketName}"/"${literature.releaseNumber}
+  pathInputsLiterature = ${literature.bucketPathBase}"/input/literature-inputs"
+  pathBasePlatformOutput = ${literature.bucketPathBase}"/output/etl/parquet"
+
+  // Release information
+  releaseNumber = "22.04"
+  epmcReleaseNumber = "22.01"
```

Comment:
Let's simplify this further: 

1. we only use `bucketName` once, so let's dump it
2. `bucketPathBase` shouldn't have the `releaseNumber`. PIS is going to put the inputs `gs://open-targets-pre-data-release/development/input/literature-inputs`. I'd say we want a config field `literature.input` which includes all the information in `bucketName`, `bucketPathBase`, and `pathInputsLiterature`